src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/store.ts

```diff
@@ -12,6 +13,12 @@ export type Attributes =
     | Models.AttributeString
     | Models.AttributeUrl;
 
+export function isAttributeEnum(attribute: unknown): attribute is Models.AttributeEnum {
+    return isObjectType<Models.AttributeEnum>(attribute, {
+        format: (v) => v === 'enum'
+    });
+}
+
```

Comment:
I am not sure we need this here. because this would also apply for email attribute as well. I think a simple check for `format` should be enough

Reply:
Do you mean we don't need the type guard, or that we don't need the isObjectType, and instead should only check if format exists?
If it's the latter, than the type check would be kinda false, because as you said, it could apply to Models.AttributeEmail